setup.py

```diff
@@ -80,6 +80,23 @@ def pre_post_plugin_type():
 
 
 if __name__ == "__main__":
+    requirements = ["netifaces", "aexpect", "netaddr", "simplejson",
+                    "autotest"]
+    try:
+        # Avocado-36 was called "avocado" and newer ones are called
+        # avocado-framework. Let's only include it in case it's not
+        # available.
+        from avocado import Test
```

Comment:
OK, I can change that.

Reply:
Well, I could detect `avocado-framwork` and proceed, but in case it's not present, but `avocado` is we'd have to still check whether it is our avocado, or whether it's just some other avocado module (like the django one). Using this is IMO simpler and clearer, what do you think?

Reply:
Or let's just simply remove all of it. It's <36 code...